README.md

```diff
@@ -1091,13 +1091,13 @@ You can configure yt-dlp by placing any supported command line option to a confi
 
 1. **Main Configuration**:
     * The file given by `--config-location`
-1. **Portable Configuration**: (Recommended for portable installations)
+2. **Portable Configuration**: (Recommended for portable installations)
```

Comment:
Actually, I will summarise this commit:
Change numbered list formatting, in order to resolve a few Pycharm warnings